views/forms/LoginForm/LoginForm.css

```diff
@@ -0,0 +1,7 @@
+.SignInButton_container {
```

Comment:
consistent block name (`LoginForm_submitBtnContainer`)

Reply:
the block should be the same for all elements in a component. If we extract elements to a new component, they become elements with a block name of the new component name.

Reply:
[1d6bb22](https://github.com/AMP-SCZ/dpdash/pull/468/commits/1d6bb2230345cbb84bf5aac92821f916e7aaad79)